.pre-commit-config.yaml

```diff
@@ -0,0 +1,58 @@
+minimum_pre_commit_version: "2.9.2"
```

Comment:
Maybe a few comment lines at the top explaining what this file is and what it's used for, for n00bs like me?

Reply:
Good idea.